memecity.game/Assets.h

```diff
@@ -20,7 +20,8 @@ namespace assets {
 		constexpr inline Asset DARK_BACKGROUND = "Sprites/big_black.bmp";
 	}
 	namespace spritesheets {
-		constexpr inline Asset HUMAN_MALE_1 = "Spritesheets/SpriteSheet.png";
+		static constexpr Asset HUMAN_MALE_1 = "Spritesheets/SpriteSheet.png";
+		static constexpr Asset TIN_CAN = "tin_can.png";
```

Comment:
Heb je de Asset ook toegevoegd aan het project in de desbetreffende map?

Reply:
ja inderdaad. Als het in de namespace `spritesheets` staat, hoort het ook in de folder `Spritesheets\` thuis.